modules/openy_features/openy_node/modules/openy_node_session/config/install/core.entity_form_display.node.session.default.yml

```diff
@@ -2,6 +2,7 @@ langcode: en
 status: true
 dependencies:
   config:
+    - field.field.node.session.field_meta_tags
```

Comment:
I'm not sure that this CT needs meta tags. There is no individual page for it. It's going to be available on class page as listing and will be rendered as teaser in various listings.